ApiConnector/NetworkConnector.swift

```diff
@@ -26,20 +26,21 @@ public protocol ApiConnectionType {
     var defaultHeaders: HTTPHeaders? { get }
     var defaultValidation: Alamofire.DataRequest.Validation? { get }
     
-    func request(with data: Data?, at endpoint: RouterType, headers: HTTPHeaders?) -> URLRequest
-    func requestData(with data: Data?, at endpoint: RouterType, headers: HTTPHeaders?) -> RequestType
+    func request(method: HTTPMethod?, with data: Data?, at endpoint: RouterType, headers: HTTPHeaders?) -> URLRequest
```

Comment:
To me it doesn't make sense why method is `Optional`. If you want to add default value for the parameter, do it like this:

```swift
func request(method: HTTPMethod = .get, with data: Data?, at endpoint: RouterType, headers: HTTPHeaders?) -> URLRequest
```